content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -38,23 +38,38 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
      [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative Studio
      will be able to parse the `.mlem` file to extract model metadata.
 
-4. Provide labels for your model. For example, if your model is about reviewing
+4. If the model is located on the remote storage, Iterative Studio will detect
+   it and suggest to enter path in the repository where to save the model file.
+   In this case, the following will happen:
+
+   - If the repository does not contain DVC, Iterative Studio will run
```

Comment:
> If the model is located in remote storage ...
> If the repository does not contain DVC

The combination of these conditions is confusing: remote storage is a DVC concept AFAIK. What do we mean by "remote storage" throughout this doc?

Reply:
Here, remote storage means external location (not inside the Git repo).

Reply:
Updated.